recipes/ipfshttpclient/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ipfshttpclient" %}
+{% set version = "0.6.0.post1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d777d91562b23813cf7aa17dbe362958087c3fcc8b98a16cbe42c562a7724055
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - flit-core >=2,<4
+    - multiaddr >=0.0.7
+    - requests >=2.11
+    - urllib3 >=1.24,<2.0
+    - httpx >=0.10.1,<1.0.0
```

Comment:
Are all of these needed for pip to install

Reply:
as far as I could tell they were needed during build. And as there is not really a build section for only python (or is there?) I put them there.

Reply:
Did you try the build with out them?

Reply:
Yes. At first urllib3 and httpx were not there. But it failed without. I can try again locally if you like.

Reply:
Tested again locally. It really fails without those in host. logs https://www.pastery.net/jbwvqq/ (from 'extracting download', but can also post full logs if you prefer)